### DISCUSSION TITLE
prohibited sign

### DISCUSSION THREAD

**Discussion by Rapacioli** [SafeExamBrowser/seb-win-refactoring#1030] [Q&A]
I would like to know what it means when the mouse pointer turns into a prohibition sign like the one I show. It happened in two exams and it would be very good to know if it is any indication of fraud. We are using SEB in moodle. Thank you!
![WhatsApp Image 2024-11-20 at 09 42 48](https://github.com/user-attachments/assets/2f2028cd-2ae7-4d1b-b3bd-6ae07e4723df)
![WhatsApp Image 2024-11-20 at 09 42 48](https://github.com/user-attachments/assets/6dc817da-d7de-43db-be3a-228af6bc973c)
![WhatsApp Image 2024-11-20 at 09 39 41](https://github.com/user-attachments/assets/5339438d-937b-467f-bd16-f2f194e3b75b)

**Comment by dbuechel** [comment]
This has nothing to do with SEB itself, the mouse cursor is controlled by the web application (as it looks Moodle) and most likely means that the user is not allowed to navigate backwards in the exam.

**Reply by dbuechel** [reply]
Though frankly I do not know Moodle very well...

**Reply by Rapacioli** [reply]
Thank you very much for your prompt response.
I thought maybe it was related to remote access. On some pages it mentions that there are ways to bypass the seb:
https://github.com/nxvvvv/safe-exam-browser-bypass/blob/main/README.md
https://archive.org/details/yt-pHrrVM46qT0

I'm glad if it's nothing.

Greetings
